torch/csrc/dynamo/guards.cpp

```diff
@@ -7740,32 +7952,35 @@ PyObject* torch_c_dynamo_guards_init() {
              bool contains,
              py::object key,
              py::object verbose_code_parts) -> void {
-            self.add_permitted_leaf_guard(std::make_shared<DICT_CONTAINS>(
-                self.get_root(),
-                contains,
-                std::move(key),
-                std::move(verbose_code_parts)));
+            self.add_permitted_leaf_guard(
+                std::make_shared<DICT_CONTAINS>(
+                    self.get_root(),
+                    contains,
+                    std::move(key),
+                    std::move(verbose_code_parts)));
           })
       .def(
           "add_dict_version_guard",
           [](DictGuardManager& self,
              py::object value,
              py::object verbose_code_parts) -> void {
```

Comment:
no user stack?

Reply:
I found two ``add_dict_version_guard`` def here. Should we remove this one?